packages/block/src/from-beacon-payload.ts

```diff
@@ -39,17 +39,17 @@ export type BeaconPayloadJSON = {
   receipts_root: PrefixedHexString
   logs_bloom: PrefixedHexString
   prev_randao: PrefixedHexString
-  block_number: PrefixedHexString
-  gas_limit: PrefixedHexString
-  gas_used: PrefixedHexString
-  timestamp: PrefixedHexString
+  block_number: string
+  gas_limit: string
+  gas_used: string
+  timestamp: string
```

Comment:
What is going on here ?!

Reply:
Through this work I realized that these values are actually numeric string, not 0xprefixed strings. See for e.g.

https://github.com/ethereumjs/ethereumjs-monorepo/blob/monorepo/typescript-test-files/packages/block/test/testdata/payload-slot-87335.ts

Reply:
Created a numeric string type for it

Reply:
Oh, that's super cool (so: the new type)! Again on this level, small change, but really another typing milestone in our libraries, once we have applied this consistently throughout the libraries! 🤩